Examples/Cocoa/Sources/Objective_C/Everything.m

```diff
@@ -704,7 +704,7 @@ - (instancetype)initWithModelDictionary:(NS_VALID_UNTIL_END_OF_SCOPE NSDictionar
             __unsafe_unretained id value = modelDictionary[@"int_enum"]; // Collection will retain.
             if (value != nil) {
                 if (value != (id)kCFNull) {
-                    self->_intEnum = [value integerValue];
+                    self->_intEnum = (EverythingIntEnum)[value integerValue];
```

Comment:
These casts are necessary for when warnings are enabled in the Pinterest builds.